modules/mysql/main.tf

```diff
@@ -55,6 +58,7 @@ resource "google_sql_database_instance" "default" {
     activation_policy           = var.activation_policy
     availability_type           = var.availability_type
     deletion_protection_enabled = var.deletion_protection_enabled
+    connector_enforcement       = local.connector_enforcement
```

Comment:
it is not True or False in the API, so using a local will allow us to convert the boolean to the necessary values hiding the complexity from the customer.

```Option values include the following: NOT_REQUIRED (Cloud SQL instances can be connected without Cloud SQL Connectors) and REQUIRED (Only allow connections that use Cloud SQL Connectors)```

Reply:
The usage of locals is only useful when 
- the local is being used in multiple places
- the logic is fairly complex for the attribute being set

None of these apply here and IMO it will actually increase readability for the consumer since they'll be able understand how this attribute is set without having to scroll back and forth.

Either way its not a huge deal; we try to make things as readable as possible with all modules, hence the comment. Feel free to resolve this and merge if you don't want to do this right now.